libs/python/detect.go

```diff
@@ -25,15 +25,32 @@ func DetectExecutable(ctx context.Context) (string, error) {
 	// the parent directory tree.
 	//
 	// See https://github.com/pyenv/pyenv#understanding-python-version-selection
+
+	// On Windows when virtualenv is created, the <env>/Scripts directory
+	// contains python.exe but no python3.exe. However, system python does have python3 entry
+	// and it is also added to PATH, so it is found first.
```

Comment:
Could this regress the non-venv cases where `python.exe` resolves to a system-wide Python 2 installation?

Reply:
Maybe, if they somehow managed to have Python2 installed in the first place, but why would they have that?

Reply:
I see python3 mentioned in another place in this module - it also won't work.

Reply:
Does it matter which Windows terminal is being used? Some users might use Git Bash for example and maybe the behaviour is different?

Reply:
that should not matter